pyverdict/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+import os
+
```

Comment:
Include the logic that compiles the java project, and copy the compiled jar file into the `lib` directory.